http-kernel-fixtures/sub-folder/cookie_page4.php

```diff
@@ -0,0 +1,19 @@
+<?php
+    $resp = new Symfony\Component\HttpFoundation\Response();
+    if (method_exists('Symfony\Component\HttpFoundation\Cookie', 'create')) {
+        $cookie = Symfony\Component\HttpFoundation\Cookie::create('srvr_cookie', 'srv_var_is_set', 0, '/sub-folder/');
+    } else {
+        $cookie = new Symfony\Component\HttpFoundation\Cookie('srvr_cookie', 'srv_var_is_set', 0, '/sub-folder/');
+    }
+    $resp->headers->setCookie($cook);
```

Comment:
You're using the non-existing `$cook` variable here. Likely it should be the `$cookie` variable defined above.

P.S.
Please run this test locally as CI would have executed it to see if it works as expected.